app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="sign_up_container">
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
```

Comment:
- [ ] kindly make your design match the wireframe
your design :point_down: 
![image](https://github.com/Enoisong/blog_app/assets/66097371/61f83f26-de76-4e2a-bf53-84a308da291f)
wireframe given :point_down: 
![Screenshot from 2023-09-06 19-26-16](https://github.com/Enoisong/blog_app/assets/66097371/0ed175c8-551c-49d6-aed6-ad210af8e5a2)